src/bin/nydusd/main.rs

```diff
@@ -375,13 +375,18 @@ fn process_fs_service(
     let bootstrap = args.value_of("bootstrap");
     // safe as virtual_mountpoint default to "/"
     let virtual_mnt = args.value_of("virtual-mountpoint").unwrap();
+    // auth for registry
+    let auth = std::env::var("IMAGE_PULL_AUTH").ok();
+
+    info!("[abin] auth: {auth:?}");
```

Comment:
Please help to remove this info message, which may leak secret information through log files.

Reply:
Done. This message is for testing purposes only.